examples/4337-passkeys/src/hooks/useNativeTokenBalance.ts

```diff
@@ -40,7 +40,10 @@ function useNativeTokenBalance(provider: ethers.Eip1193Provider, address: string
       const pollInterval = opts?.pollInterval || 5000
 
       updateBalance()
-      const interval = setInterval(updateBalance, pollInterval)
+      // We convert the interval to a number because typescript thinks this call is ambiguous
+      // since node.js and browser have different return types for setInterval
+      // more info: https://github.com/Microsoft/TypeScript/issues/30128#issuecomment-807394387
+      const interval = +setInterval(updateBalance, pollInterval)
```

Comment:
Do you know why this just started manifesting now?

Reply:
> Do you know why this just started manifesting now?

Well, looks like, in the end, it was the typescript version somehow 🙈 Probably `pnpm` shenanigans...

I found a better way to fix it, though, I created a new vite app through their CLI tool and copied the update typescript configs from there and that fixed the issue